cumulus/steps/ec2/dns.py

```diff
@@ -1,47 +1,43 @@
-from troposphere import (
-    Ref, Join)
+from troposphere import Join
 from troposphere import route53
 
 from cumulus.chain import step
 
+DNS_NAME = 'AlbAlias%s'
+
 
 class Dns(step.Step):
 
     def __init__(self,
-                 namespace,
+                 dns_name,
                  base_domain,
                  hosted_zone_id,
-                 dns_name,
+                 target,
                  ):
+        step.Step.__init__(self, name='Dns')
 
-        step.Step.__init__(self)
-
+        self.target = target
         self.base_domain = base_domain
-        self.namespace = namespace
         self.hosted_zone_id = hosted_zone_id
         self.dns_name = dns_name
 
     def handle(self, chain_context):
         template = chain_context.template
 
-        name = 'AlbAlias%s' % chain_context.instance_name
-
         template.add_resource(route53.RecordSetGroup(
-            "Route53Records",
+            "Route53Record",
```

Comment:
This name seems interesting..  we may just be adding a single recordset but the resource is a group that can take multiple.